articles/quickstart/backend/java-spring-security5/interactive.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Spring Boot API
+description: Secure your API using Spring Security 5 and Auth0
+interactive: true
+alias:
+  - spring security
+  - spring
+topics:
+  - quickstart
+  - backend
+  - spring
+files:
+  - files/application
+  - files/audience-validator
+  - files/security-config
+  - files/message
+  - files/api-controller
+github:
+  path: 01-Authorization-MVC
+---
+
+# Secure your API using Spring Security 5 and Auth0
+
+Auth0 allows you to quickly add authentication and gain access to user profile information in your application. This guide demonstrates how to integrate Auth0 with any new or existing Spring Boot application using the [auth0-spring-security-api](https://github.com/auth0/auth0-spring-security-api) package.
```

Comment:
Is it expected that this SDK isn't installed as a dependency or used in any of the code snippets?

Reply:
I am not sure what we are expecting here Fred, we already have a code snippet that shows how to add the dependency